### PR TITLE
Angular Initialization Formatting

### PR DESCRIPTION
**Pull request by nakhbari**
* Initialization code should be separated by new lines and indented with 4 spaces
  * We add comments to the end of the lines to help formatting with the Eclipse IDE
```javascript
  angular//
      .module(...)//
      .controller(...);
```

### CODE REVIEW

**Comment by dbradicich** [comment]
+0 I like how it looks, but manual formatting isn't very exciting, I suppose copy/paste will usually make this simple enough, I'll defer to others

File: Kaos-Frontend-Coding-Standards.md [hunk 1]
```diff
@@ -85,6 +85,13 @@ At Sonatype we value the stability and maintainability of the code base while st
   * This enforces the separation of view and controller
   * Performance concerns can be mitigated by compiling the markup into Javascript during the build
   * HTML in Javascript is difficult to read and maintain
+* Initialization code should be separated by new lines and indented with 4 spaces
+  * We add comments to the end of the lines to help formatting with the Eclipse IDE
+```javascript
+  angular//
+      .module(...)//
+      .controller(...);
+```
```
Comment (mpiggott) [inline Kaos-Frontend-Coding-Standards.md]: I'm still -*1*

**Comment by whyjustin** [comment]
+1

File: Kaos-Frontend-Coding-Standards.md [hunk 2]
```diff
@@ -85,6 +85,13 @@ At Sonatype we value the stability and maintainability of the code base while st
   * This enforces the separation of view and controller
   * Performance concerns can be mitigated by compiling the markup into Javascript during the build
   * HTML in Javascript is difficult to read and maintain
+* Initialization code should be separated by new lines and indented with 4 spaces
+  * We add comments to the end of the lines to help formatting with the Eclipse IDE
+```javascript
+  angular//
```
Comment (whyjustin) [inline Kaos-Frontend-Coding-Standards.md]: I prefer a space between the EOL and comment

**Comment by nakhbari** [comment]
As per our discussion [1], this proposal is denied and the developer can use whichever initialization style they prefer. 

[1]https://docs.google.com/spreadsheets/d/1eFKyXJsOPDrqxppQQXzwGpuG8CZV33h9OYu2M5GTPv0/edit#gid=0

**Pull request closed without merging**